pkg/manager/member/tiflash_failover.go

```diff
@@ -14,104 +14,75 @@
 package member
 
 import (
-	"fmt"
-	"time"
-
 	"github.com/pingcap/tidb-operator/pkg/apis/pingcap/v1alpha1"
 	"github.com/pingcap/tidb-operator/pkg/controller"
-	"github.com/pingcap/tidb-operator/pkg/util"
-	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/uuid"
-	"k8s.io/klog/v2"
 )
 
-// TODO reuse tikvFailover since we share the same logic
 type tiflashFailover struct {
```

Comment:
Is it possible if we eliminate `tiflashFailover` and use `sharedStoreFailover` instead? Maybe just keep the implementation of `tiflashStoreAccess` in this file?